### ISSUE TITLE
Forms seem to be broken for Voice Control with iOS

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#22749]
## What happened?
When I try to use Voice Control with forms, I'm unable to get into edit mode.  There was a change recently made so that there were duplicate elements that showed up with Talkback / Voice Over and I think this may have broken Voice Control with iOS.  See screen shots.

## Specs:
- Device: iPhone XR with Voice Control enabled.

## Steps to Reproduce
With Voice Control, try to select fields in forms.

## Desired behavior
Voice Control can select fields in forms for editing.

## Acceptance Criteria
Voice Control can select fields in forms for editing.

## Bug Severity
- 3- Medium

## Linked to Story
n/a - regression

## Screen shot(s) and additional information

Examples of forms that can not be accessed with Voice Control.

![image.png](https://images.zenhubusercontent.com/5f6a83c84093730d05b5cee6/25a06218-d990-4d69-85eb-9010d17a1b5b)

**Comment by kxiao133** [comment]
For QA, it would be good to do a full accessibility regression pass around forms. Wrapping components in pressables sometimes interferes with the text input accessibility.

 I didn't notice any new bugs when testing with VoiceControl and VoiceOver on my tester device or on the Android simulator, but we should probably double check

**Comment by digitalken** [comment]
PASS 6/17/2021 - Looks good.  Gave forms a quick run through on iOS with VoiceControl.  Looks good.  Will probably do more regression on forms with voice control.  closing.
